data/secscan_model/secscan_v4_model.py

```diff
@@ -163,6 +170,10 @@ def eligible_manifests(base_query):
         if max_id is None or min_id is None or min_id > max_id:
             return None
 
+        can_reindex = (datetime.utcnow() - ManifestSecurityStatus.last_indexed) > timedelta(
```

Comment:
This won't work efficiently on MySQL because it'll have to compute for each row, rather than using the index; instead, do this:

```
can_reindex_threshold = datetime.utcnow() - timedelta(....)
can_reindex = (ManifestSecurityStatus.last_indexed <= can_reindex_threshold)
```